### ISSUE TITLE
Some material icons doesn't exist in my SDK

### ISSUE THREAD

**Issue by SuhwanCha** [flutter/flutter#95463]
File: packages/flutter/lib/src/material/icons.dart

My Flutter Sdk, icons.dart has some difference with icons.dart on master branch

## My SDK
```dart
  /// <i class="material-icons-outlined md-36">edit_off</i> &#x2014; material icon named "edit off" (outlined).
  static const IconData edit_off_outlined = IconData(0xf00c, fontFamily: 'MaterialIcons');

  /// <i class="material-icons md-36">edit_road</i> &#x2014; material icon named "edit road".
  static const IconData edit_road = IconData(0xe220, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-sharp md-36">edit_road</i> &#x2014; material icon named "edit road" (sharp).
  static const IconData edit_road_sharp = IconData(0xe91b, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-round md-36">edit_road</i> &#x2014; material icon named "edit road" (round).
  static const IconData edit_road_rounded = IconData(0xf6fa, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-outlined md-36">edit_road</i> &#x2014; material icon named "edit road" (outlined).
  static const IconData edit_road_outlined = IconData(0xf00e, fontFamily: 'MaterialIcons');

  /// <i class="material-icons md-36">eject</i> &#x2014; material icon named "eject".
  static const IconData eject = IconData(0xe221, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-sharp md-36">eject</i> &#x2014; material icon named "eject" (sharp).
  static const IconData eject_sharp = IconData(0xe91d, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-round md-36">eject</i> &#x2014; material icon named "eject" (round).
  static const IconData eject_rounded = IconData(0xf6fc, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-outlined md-36">eject</i> &#x2014; material icon named "eject" (outlined).
  static const IconData eject_outlined = IconData(0xf00f, fontFamily: 'MaterialIcons');
```

## On Github's master branch
```dart
  static const IconData edit_road_rounded = IconData(0xf6fa, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-outlined md-36">edit_road</i> &#x2014; material icon named "edit road" (outlined).
  static const IconData edit_road_outlined = IconData(0xf00e, fontFamily: 'MaterialIcons');

  /// <i class="material-icons md-36">egg</i> &#x2014; material icon named "egg".
  static const IconData egg = IconData(0xf04f8, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-sharp md-36">egg</i> &#x2014; material icon named "egg" (sharp).
  static const IconData egg_sharp = IconData(0xf0404, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-round md-36">egg</i> &#x2014; material icon named "egg" (round).
  static const IconData egg_rounded = IconData(0xf0311, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-outlined md-36">egg</i> &#x2014; material icon named "egg" (outlined).
  static const IconData egg_outlined = IconData(0xf05f2, fontFamily: 'MaterialIcons');

  /// <i class="material-icons md-36">egg_alt</i> &#x2014; material icon named "egg alt".
  static const IconData egg_alt = IconData(0xf04f7, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-sharp md-36">egg_alt</i> &#x2014; material icon named "egg alt" (sharp).
  static const IconData egg_alt_sharp = IconData(0xf0403, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-round md-36">egg_alt</i> &#x2014; material icon named "egg alt" (round).
  static const IconData egg_alt_rounded = IconData(0xf0310, fontFamily: 'MaterialIcons');

  /// <i class="material-icons-outlined md-36">egg_alt</i> &#x2014; material icon named "egg alt" (outlined).
  static const IconData egg_alt_outlined = IconData(0xf05f1, fontFamily: 'MaterialIcons');

  /// <i class="material-icons md-36">eject</i> &#x2014; material icon named "eject".
  static const IconData eject = IconData(0xe221, fontFamily: 'MaterialIcons');
```

## Issues
There isn't **egg** Icons on my SDK, and I found there isn't egg data in my SDK.
How can I figure it out?

Also, I tried 
```dart
Icon(
    IconData(0xf05f1, fontFamily: 'MaterialIcons'),
)
```
but it still doesn't work.

My flutter version
```
% flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.8.0, on macOS 12.0.1 21A559 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 13.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.2)
[✓] VS Code (version 1.63.0)
[✓] Connected device (2 available)
    ! Error: Suhwan’s iPhone is busy: Preparing the watch for development via Suhwan’s iPhone. Xcode will continue when Suhwan’s iPhone is finished. (code -10)

• No issues found!
```

**Comment by maheshmnj** [comment]
Hi @SuhwanCha, Thanks for filing the issue. it looks like you are on a stable channel 2.8.0 and comparing it with master channel 2.9.0-1. I would advise you to please switch to latest master channel. You can do this by running

```
flutter channel master
flutter upgrade
```

FYI, A recent PR introduced 1028 new material icons https://github.com/flutter/flutter/pull/95007 so that must be worth checking out.

**Comment by maheshmnj** [comment]
To know more about the flutter channels please refer this wiki https://github.com/flutter/flutter/wiki/Flutter-build-release-channels

**Comment by SuhwanCha** [comment]
Hi @maheshmnj ,

Thank you for helping me.
I misunderstood that it is released with 2.8.0.

Thanks!
